pwiz_tools/Skyline/Model/Proteome/ProteinAssociation.cs

```diff
@@ -94,6 +94,10 @@ public void UseFastaFile(string file, Func<FastaSequence, IEnumerable<Peptide>>
                     }
                 }
             }
+            catch (InvalidDataException)

```

Comment:
I would be inclined to get rid of this whole try/catch block, and just let all exceptions bubble up. The wrapping with "InvalidDataException" a few lines later doesn't appear to add much information to the error message.